### ISSUE TITLE
Overrides in groups tab

### ISSUE THREAD

**Issue by sanmont3drepo** [3drepo/3drepo.io#4257]
### Description
In the groups tab we need to be able to set groups overrides. This depends on #4224 

### Goals
- [ ] Have overrides in the groups tab

### Tasks
- [ ] When clicking a collection with overrides it should generate the override for all the descendants
- [ ]

**Comment by carmenfan** [comment]
Merged into #3933
